src/main/fc/rc_modes.c

```diff
@@ -104,6 +104,16 @@ void updateMasksForStickyModes(const modeActivationCondition_t *mac, boxBitmask_
     }
 }
 
+void updateMasksForLinkedMac(const modeActivationCondition_t *mac, boxBitmask_t *andMask, boxBitmask_t *newMask)
+{
+    bool bAnd = (mac->modeLogic == MODELOGIC_AND) || bitArrayGet(andMask, mac->modeId);
+    bool bAct = bitArrayGet(andMask, mac->linkedTo) != bitArrayGet(newMask, mac->linkedTo);
+    if (bAnd)
+        bitArraySet(andMask, mac->modeId);
```

Comment:
As per coding style convention (https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md#curly-braces), please always use blocks after `if`, `else`.

Reply:
No problem. This was a copy of its brother function `updateMasksForMac`. I'll correct it in both places.